spec/std/big/big_decimal_spec.cr

```diff
@@ -482,6 +482,229 @@ describe BigDecimal do
     it { -2.91.to_big_d.trunc.should eq(-2) }
   end
 
+  describe "#round" do
+    describe "rounding modes" do
+      it "to_zero" do
+        "-1.5".to_big_d.round(:to_zero).should eq -1.0.to_big_d
```

Comment:
For sanity, I'd suggest to use either int or string literals for the expected values as well (away with those nasty float literals!).